pages/common/docker-build.md

```diff
@@ -15,6 +15,10 @@
 
 `docker build --tag {{name:tag}} .`
 
+- Build a docker image with no build context:
+
+`cat {{Dockerfile}} | docker build --tag {{name:tag}} -`
```

Comment:
Could you use a shell redirect instead?
```suggestion
`docker build --tag {{name:tag}} - < {{Dockerfile}}`
```

Reply:
That's what I had originally, but Windows (CMD/PowerShell) doesn't support that.

Reply:
A lot of the common pages don't work for windows. But they work for Mac and Linux for example.

Reply:
Fair enough, I'd still say it's nicer to keep things cross-platform where possible, though.
I'd use the `<` syntax day-to-day, however when I'm writing documentation or tutorials where I think Windows users will consume it, I think it's better to use `cat {} |`.

Otherwise, I think it'd be better to duplicate this page to Windows with the Windows specific syntax.

Reply:
I agree, was just giving the point.

Reply:
Do you have documentation for that? The `command < path/to/file` syntax *does* actually work on Windows as far as I'm aware. Not sure about Powershell, but it definitely works in CMD I'm pretty sure.

Reply:
Ahh, I only tested in PowerShell when I tried it on Windows. 🤔 
If I can get confirmation that it works in CMD, then I'll concede.

I'll try later, but if someone else uses Windows and can drop a comment here, that'd be nice!

Reply:
PowerShell ❌ 
cmd.exe ✅